### ISSUE TITLE
Error running transpile-raw task: Module does not export 'moment'

### ISSUE THREAD

**Issue by flibX0r** [moment/moment#2394]
Fresh clone of Build 2.10.3 (25cc720fbaab7323a1016cd0beb6a36e349c41be)

Grunt output:
````
C:\Dev\moment>grunt
Running "jshint:all" (jshint) task
>> 305 files lint free.

Running "jscs:all" (jscs) task
>> 305 files without code style errors.

Running "transpile" task

Running "clean:build" (clean) task
>> 1 path cleaned.

Running "transpile-raw" task
>> build/umd/moment.js
>> build/umd/locale/*.js
>> build/umd/test/moment/*.js
>> error transpiling [Error: Module 'C:/Dev/moment/src/moment' does not export 'moment' (imported by 'test/locale/af')]
Warning: Module 'C:/Dev/moment/src/moment' does not export 'moment' (imported by 'test/locale/af') Use --force to continue.

Aborted due to warnings.
````

The specific locale that triggers the error seems non-deterministic and changes on each run.

System is Windows 7, node v0.12.4, fresh install of all dependencies

**Comment by raleksandar** [comment]
Same problem with eef69fc which I just forked of, can't run tests due to this problem (will try to fix and submit PR)

**Comment by raleksandar** [comment]
Just created a PR which addresses this issue.

But I still had problems running grunt tasks since test task was failing with `Warning: utils_extend__extend is not defined` error. But it's unrelated to this issue since it's probably a regression in esperanto module as downgrading it to 0.6.9 (all 0.7.x version I've tried had this problem) makes grunt tasks run (I'm leaving this note here just in case someone else experiences the same problem).

**Comment by philjohnson789** [comment]
Thanks, I was also experiencing this problem on Windows 7 (but not Max OS or Linux). After merging your changes from #2439, I was finally able to run all of the unit tests after changing the esperanto dependency in package.json.

This worked perfectly for me: `"esperanto": "^0.6",`
And this version failed while running qtest: `"esperanto": "^0.7",`

**Comment by ichernev** [comment]
So is this fixed with the merge of #2439?
@raleksandar @flibX0r @philjohnson789

**Comment by flibX0r** [comment]
Works for me

**Comment by ichernev** [comment]
Fixed in #2439.
